### ISSUE TITLE
Material.CREAKING_HEAR itemstack exception

### ISSUE THREAD

**Issue by GliczDev** [MockBukkit/MockBukkit#1237]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest version of MockBukkit?

- [X] I am using the latest version of MockBukkit.

### Minecraft Version

1.21

### Describe the bug

```
org.mockbukkit.mockbukkit.exception.ItemMetaInitException: Failed to instanciate item meta class class org.mockbukkit.mockbukkit.inventory.meta.BlockStateMetaMock
	at org.mockbukkit.mockbukkit.inventory.ItemStackMock.findItemMeta(ItemStackMock.java:406)
	at org.mockbukkit.mockbukkit.inventory.ItemStackMock.<init>(ItemStackMock.java:64)
	at org.mockbukkit.mockbukkit.inventory.ItemTypeMock.createItemStack(ItemTypeMock.java:157)
	at org.mockbukkit.mockbukkit.inventory.ItemTypeMock.createItemStack(ItemTypeMock.java:151)
	at me.glicz.skanalyzer.SkAnalyzer.main(SkAnalyzer.java:68)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:74)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
	at org.mockbukkit.mockbukkit.inventory.ItemStackMock.findItemMeta(ItemStackMock.java:398)
	... 4 more
Caused by: java.lang.UnsupportedOperationException: 'CREAKING_HEART' is not known to have a BlockStateMeta ItemMeta type
	at org.mockbukkit.mockbukkit.inventory.meta.BlockStateMetaMock.<init>(BlockStateMetaMock.java:135)
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	... 7 more
```

### Reproducible Test

```java
MockBukkit.mock();
Material.CREAKING_HEART.asItemType().createItemStack();
```

**Comment by 4everTheOne** [comment]
Hello @GliczDev , thank you for this report. The issue is resolved in [v4.20.1](https://github.com/MockBukkit/MockBukkit/releases/tag/v4.20.1) could you please validate?

Thanks in advance.

**Comment by GliczDev** [comment]
yeah it works, thanks
